website/docs/terms/data-lake.md

```diff
@@ -0,0 +1,112 @@
+
+### Exploratory data analysis (EDA)
+
+Because you don’t need to impose a formal structure for how data is organized in a data lake, you can perform preliminary data exploration on that data, such as calculate summary statistics, discover anomalies and outliers, and plot data visualizations to derive preliminary insights. Commonly referred to as EDA, this is typically conducted as an initial step before formalizing a data science or machine learning use case. 
+
+## Data lake vs. data warehouse
+
+|  |  Data lake | Data warehouse |
+|---|---|---|
+| Types of data  | Structured, Semi-Structured, Unstructured | Structured  |
+| Data stored in  | Folders and files in raw format | Schemas and tabular data format |
+| Schema/schema definition | Store data in its raw format, transform the data later | Must know upfront |
+| Intended users | Data engineers, Analytics engineers, Data analysts,  Data scientists | Analytics engineers, Data analysts,  Business analysts |
+| Common use cases | Data archiving and storage, data science and machine learning,exploratory data analysis | Business intelligence,dashboarding,reporting and analytics  |
+
+## Data platforms that support data lake workloads
+
+|  Data Platform | Description | Data warehouse |
+|:---:|:---:|---|
+| Cloudera | Cloudera Open Data Lakehouse is a platform that provides data lake flexibility and data warehouse performance in a single platform.  | Structured  |
+| Databricks | Databricks is a cloud-based collaborative data science, data engineering, and data analytics platform that brings the best of data warehouses and data lakes into a single unified platform.  | Schemas and tabular data format |
+| Dremio | Dremio is the data lakehouse platform built for SQL and built on open source technologies that both data engineers and data analysts love. Dremio powers BI dashboards and analytics directly on data lake storage. | Must know upfront |
+| Snowflake | Snowflake is a fully-managed platform for data warehousing, data lakes, data engineering, data science, and data application development. | Analytics engineers, Data analysts,  Business analysts |
+
+## Conclusion
+
+The data lake is the younger data management platform compared to its data warehouse counterpart. Because of its unique ability to hold large amounts of data in its native, raw format, it has allowed organizations to store all its data in a centralized place, even if sometimes they don’t have a definitive use case for the data yet. In addition, it serves as a great buffer and landing zone for data before it is ultimately transformed and aggregated in a data warehouse. Lastly, it has unlocked a world of new possibilities by enabling organizations to build data science and machine learning use cases on top of it. The data lake is an integral pillar in the Modern Data Stack and the practice of analytics engineering.
+
+## Additional reading
+- [Glossary: Data warehouse](https://docs.getdbt.com/terms/data-warehouse) 
```

Comment:
@johnblust and @KiraFuruichi, apologies about the drive by comment, but I'd love to see us use relative links, especially if we're going to change the domain at some point.
```suggestion
- [Glossary: Data warehouse](/terms/data-warehouse) 
```